src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISKOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISKOUNT = 20;
```

Comment:
There is a typo in the variable name SHORT_TERM_DISKOUNT. It should be SHORT_TERM_DISCOUNT.